src/grid/ref/grid_ref_collint.h

```diff
@@ -66,6 +66,7 @@ ortho_cx_to_grid(const int lp, const int k1, const int k2, const int j1,
 #if (GRID_DO_COLLOCATE)
     // collocate
     double reg[4] = {0.0, 0.0, 0.0, 0.0};
+#pragma omp simd reduction(+ : reg)
```

Comment:
The REf backend is now indeed only a reference implementation. The default CPU backend is now in:

https://github.com/cp2k/cp2k/tree/master/src/grid/cpu

Reply:
I'd like to keep the reference backend as simple as possible.

However, your fanzy directives are very much appreciated in the optimized CPU backend ;-)

https://github.com/cp2k/cp2k/blob/39d9274f51dfd4eb0249496921a785bb611b0c44/src/grid/cpu/grid_cpu_collint.h#L49

https://github.com/cp2k/cp2k/blob/39d9274f51dfd4eb0249496921a785bb611b0c44/src/grid/cpu/grid_cpu_collint.h#L64

Reply:
Ahh, interesting point. Will do this asap.